pkg/resources/database_role.go

```diff
@@ -1,21 +1,16 @@
 package resources
 
 import (
-	"bytes"
+	"context"
 	"database/sql"
-	"encoding/csv"
 	"fmt"
 	"log"
-	"strings"
 
-	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/snowflake"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/helpers"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-const (
-	databaseRoleIDDelimiter = '|'
-)
-
 var databaseRoleSchema = map[string]*schema.Schema{
```

Comment:
again, isn't this schema missing some computed attributed like `created_at`, `is_default` and `is_current`?

Reply:
Same as above.